lib/python/rose/bush.py

```diff
@@ -741,13 +743,53 @@ def _get_user_suite_dir_root(self, user):
             self._get_user_home(user),
             self.bush_dao.SUITE_DIR_REL_ROOT))
 
+    @staticmethod
+    def _check_string_for_path(string):
+        """Raise HTTP 403 error if the provided string contain path chars.
+
+        Raises:
+            cherrypy.HTTPError(403)
+
+        """
+        temp = os.path.split(string)
+        if len(temp) > 2 or (len(temp) == 2 and temp[0] != ''):
```

Comment:
`temp` is always a pair according to docs? https://docs.python.org/2/library/os.path.html#os.path.split